kong.conf.default

```diff
@@ -1124,6 +1124,29 @@
                                  # Detailed discussion of this behavior is
                                  # available in the online documentation.
 
+#pg_keepalive_timeout = 60000    # Specify the maximal idle timeout ( in ms )
```

Comment:
```suggestion
#pg_keepalive_timeout = 60000    # Specify the maximal idle timeout (in ms)
```

Reply:
Does it make sense to make the default value to be `nil` (default OpenResty behavior)?

> The first optional argument, timeout, can be used to specify the maximal idle timeout (in milliseconds) for the current connection. If omitted, the default setting in the [lua_socket_keepalive_timeout](https://github.com/openresty/lua-nginx-module#lua_socket_keepalive_timeout) config directive will be used. If the 0 value is given, then the timeout interval is unlimited.

Reply:
Yes makes sense, updated